tests/TestAsset/VersionIsIdentical.php

```diff
@@ -34,16 +34,16 @@ public function __construct(
         );
     }
 
-    protected function matches($constraint) : bool
+    protected function matches($version) : bool
     {
-        /* @var $constraint Version */
+        /* @var $version Version */
```

Comment:
Why is the type not constrained in the signature?

Reply:
@Ocramius `Declaration should be compatible with Constraint->matches()` warning.